### PR TITLE
Rfc rc1

### PR DESCRIPTION
**Pull request by nmanthey**
*Issue #, if available:*

*Description of changes:*

 * disable gotocc as default
 * get rid of configuration directory
 * fix existing tests
 * add tests to include one-line-cr-bot
 * enable regression tests during CI

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: backends/cbmc/gotocc-wrapper.sh [hunk 1]
```diff
@@ -258,13 +258,16 @@ function checkIncludes
       local inc=$(sed -n "${l}p" "$f" | \
                   sed -e 's/^.*#include[[:space:]]*[<"]//' \
                       -e 's/[>"][[:space:]]*$//')
-      if $GOTO_DIFF_BINARY -u "$tmpfile.o" "${tmpfile}-i.o" | grep -q "^[+-]"
+      if [ "$GOTO_DIFF_BINARY" != "$XX/usr/bin/goto-diffXX" ]
```
Comment (tautschnig) [inline backends/cbmc/gotocc-wrapper.sh]: Spurious `$` before `XX/usr/bin/goto-diffXX`

**Review by tautschnig — APPROVED** [review]
Looks good to me except for one spurious `$` to be removed.

File: backends/cbmc/gotocc-wrapper.sh (hunk 1 above)
Comment (nmanthey) [reply backends/cbmc/gotocc-wrapper.sh]: True, will fix.

**Comment by nmanthey** [comment]
Main diff of the update: rebased on top of the other PR, fixed above nit. Fixed another issue related to goto-ld handling when using the CBMC package that does not bring goto-ld. Added documentation update.

**Pull request merged**